core/src/main/java/com/orientechnologies/orient/core/security/OTokenSignImpl.java

```diff
@@ -135,14 +134,7 @@ public String getDefaultKey() {
     if (configKey != null && configKey.length() > 0) key = Base64.getUrlDecoder().decode(configKey);
 
     if (key == null) {
-      try {
-        key =
-            OSecurityManager.digestSHA256(
-                String.valueOf(SecureRandom.getInstanceStrong().nextLong()));
-      } catch (NoSuchAlgorithmException e) {
```

Comment:
Is there a reason we have taken away the try catch?  It seems like NoSuchAlgorithmException won't actually be thrown here I guess.  However looking at the code for digestSHA256, it looks like a OConfigurationException could be thrown.  Should we catch that, still, and re-throw as before?

Reply:
Only NoSuchAlgorithmException was caught here before, and is no longer possible, so removed handler for it.